Formula/libunwind.rb

```diff
@@ -4,11 +4,14 @@ class Libunwind < Formula
   url "https://download.savannah.nongnu.org/releases/libunwind/libunwind-1.5.0.tar.gz"
   sha256 "90337653d92d4a13de590781371c604f9031cdb50520366aa1e3a91e1efb1017"
   license "MIT"
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, x86_64_linux: "01d88381315b99c187e8351a10764ff2e38872bd178fe51bf3ddedba3d6418ee"
   end
 
+  keg_only "it conflicts with `llvm`"
```

Comment:
This really is the problem -- `julia` depends on both `llvm` and `libunwind` on Linux. I think Julia will eventually use LLVM's `libunwind` (it does on the master branch for macOS). But until then I think we'll have to drop the `libunwind` dependency and just use the vendored one, which is installed in a private location.